### PR TITLE
Create SIMD-09-30.md

### PR DESCRIPTION
**Pull request by penzn**

### CODE REVIEW

**Comment by penzn** [comment]
I don't seem to be able to add reviewers. @conrad-watt wanted to review the summary comment. Also @dtig, @yurydelendik, @justinmichaud, @Maratyszcza, @abrown, please take a look.

**Review by Maratyszcza — APPROVED** [review]

**Comment by penzn** [comment]
@dtig did you get any better transcription?

In case there are no further changes, I think we should merge this.

By the way, on the issue of adding reviewers, I think Github has a contributor role without write access, that should allow for requesting reviews. It would be good if all subgroup (co)chairs had at least that role for this repo ;)  (/CC @dschuff)

**Comment by penzn** [comment]
I made some minor changes, I think it should be ready to merge.

**Review by ngzhian — CHANGES_REQUESTED** [review]

File: simd/2022/SIMD-09-30.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+### Finalizing the specification for Phase 4 vote
+
+CW: want to discuss non-determinism, float idea of deterministic FMA
+
+DG, MD: we would have to discuss that later, Zhi is out, also sounds he might have addressed it
+
+DG: anything else we need to consider?
+
+MD: Dood on my side. I implemented optimizations using relaxed-simd in xnnpack, got very good speedup
```
[the remark this answers is not in the document]
Comment (penzn) [reply simd/2022/SIMD-09-30.md:49]: ~Dood~Good catch

**Comment by penzn** [comment]
Thanks for review @ngzhian, also caught a couple more typos after

**Comment by penzn** [comment]
ping @ngzhian

**Review by ngzhian — APPROVED** [review]

**Pull request merged**